Update Kapacitor's "Getting Started" examples to use Telegraf 0.10.0 schema

rossmcdonald opened the pull request:
Fixes #185

rkuchan commented:
The query under "There should be some cpu metrics in a database called `kapacitor_example`. Confirm this with this query:" should be: 

```
curl -G 'http://localhost:8086/query?db=kapacitor_example' --data-urlencode 'q=SELECT count(usage_idle) FROM cpu'
```

rossmcdonald commented:
@rkuchan Thanks, I added a fix for the curl command.

I also removed the misleading section about the debug output. The wording may need to be fixed now, but hopefully it shouldn't cause any more confusion.

nathanielc commented on content/kapacitor/v0.10/introduction/getting_started.md:
```diff
@@ -69,20 +62,18 @@ The following is a simple Telegraf configuration file that will send just cpu me
     percpu = false
     totalcpu = true
-    drop = ["cpu_time"]
+    drop = ["time_*"]
```
These changes look good, do you want me to add then to #194?

rossmcdonald replied:
@nathanielc Sure.

rossmcdonald commented:
Closing in favor of #194
